SonarQube.Common/Interfaces/ILogger.cs

```diff
@@ -12,10 +12,19 @@ namespace SonarQube.Common
     /// </summary>
     public interface ILogger
     {
-        void LogMessage(string message, params object[] args);
+        void LogMessage(LoggerVerbosity verbosity, string message, params object[] args);
```

Comment:
The interface doesn't need "LogMessage", "LogDebug" and "LogInfo"; either LogMessage could be an extension method, or LogDebug and LogInfo could be. It doesn't matter much which, but it you are never calling LogMessage directly from outside then I would drop that method and leave the other two.